internal/civisibility/integrations/gotesting/testing_test.go

```diff
@@ -117,7 +120,7 @@ func TestMain(m *testing.M) {
 	testSpans := getSpansWithType(finishedSpans, constants.SpanTypeTest)
 	fmt.Printf("Number of tests received: %d\n", len(testSpans))
 	showResourcesNameFromSpans(testSpans)
-	if len(testSpans) != 36 {
+	if len(testSpans) != 37 {
 		panic("expected exactly 36 test spans, got " + strconv.Itoa(len(testSpans)))
```

Comment:
We also need to update the string here and :92 to `37`

Reply:
good catch, done in [453eb0c](https://github.com/DataDog/dd-trace-go/pull/2910/commits/453eb0cca57a01fc607abcfdc7272834230d1927)